editor/plugins/control_editor_plugin.cpp

```diff
@@ -650,6 +650,10 @@ void SizeFlagPresetPicker::_preset_button_pressed(const int p_preset) {
 	emit_signal("size_flags_selected", flags);
 }
 
+void SizeFlagPresetPicker::_expand_button_pressed() {
+	emit_signal("expand_toggled", expand_button->is_pressed());
```

Comment:
I'd call it `expand_flag...` instead of `expand...` in signals and methods, so it's consistent with `size_flags...`.